File: LSC/Sample/iOS_OSX/Common/LogModule.m
//
//  LogModule.m
//  Sample
//
//  Created by admin on 16/9/22.
//  Copyright © 2016年 hy. All rights reserved.
//

#import "LogModule.h"

@implementation LogModule

+ (void)writeLog:(NSString *)message
{
    NSLog(@"** message = %@", message);
}

@end

File: LSC/Sample/iOS_OSX/Sample-OSX-Swift/LogModule.swift
//
//  LogModule.swift
//  Sample
//
//  Created by admin on 16/12/1.
//  Copyright © 2016年 hy. All rights reserved.
//

import LuaScriptCore_OSX_Swift

@objc(LogModule)
class LogModule: NSObject, LuaExportType
{
    @objc static func writeLog(message : String) -> Void
    {
        NSLog("** message = %@", message);
    }
}
